### ISSUE TITLE
Training data source

### ISSUE THREAD

**Issue by nickhir** [SexChrLab/SexInference#3]
I was wondering if you could provide more information how you have generated the training data. In the README you say that they come from the "GTEx data", but AFAIK GTEx only has ~1k individuals, whereas the training data has >17k. Furthermore, I was wondering which tissue the RNAseq data comes from or if you have just merged all the tissues. 

Thank you very much in advance!

**Comment by mathbionerd** [comment]
Yes, GTex has about 960 participants, each with multiple tissues, now up to 30,000 total samples.
